conflowgen/api/container_dwell_time_distribution_manager.py

```diff
@@ -58,3 +61,16 @@ def set_container_dwell_time_distribution(
             sanitized_distribution
         )
         DataSummariesCache.reset_cache()
+
+    def get_average_container_dwell_time(self, start_date: datetime.datetime, end_date: datetime.datetime) -> float:
+        """
+        Uses the inbound and outbound vehicle capacities service and the mode of transport input distribution
```

Comment:
Instead of a verbal mentioning of services and distributions, could you directly link to them? For reference, see the link to the `.generate()` method at https://conflowgen--187.org.readthedocs.build/en/187/api.html#conflowgen.PortCallManager or a link to a preview at https://conflowgen--187.org.readthedocs.build/en/187/api.html#conflowgen.ContainerFlowByVehicleTypePreviewReport. Then the people can just click on it and it makes the API documentation easier to use :-)

Could you also shortly comment on why we need a start and end date? For an average calculation, that might not be really straight-forward for a new user. The reason is simple though - depending on the requested time range, the number of vessel arrivals change. Theoretically, someone could have a vessel arriving every month and then the question is whether in the given time range that vessel calls the terminal or not. Maybe you can copy the text from https://conflowgen--187.org.readthedocs.build/en/187/api.html#conflowgen.InboundAndOutboundVehicleCapacityPreview - *The earliest day to consider when checking the vehicles that move according to schedules* and *The latest day to consider when checking the vehicles that move according to schedules*.

I might have given you some wrong information on the type of `start_date` and `end_date`. At https://conflowgen--187.org.readthedocs.build/en/187/api.html#conflowgen.InboundAndOutboundVehicleCapacityPreview, it is shown as `datetime.date`. Could you update that?